docs/design.html

```diff
@@ -339,9 +339,7 @@
 <p>
 <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring The Log Cleaner</a></h4>
 
-The log cleaner is disabled by default. To enable it set the server config
-  <pre>  log.cleaner.enable=true</pre>
-This will start the pool of cleaner threads. To enable log cleaning on a particular topic you can add the log-specific property
+The log cleaner is enabled by default. This will start the pool of cleaner threads. To enable log cleaning on a particular topic you can add the log-specific property
```

Comment:
Thanks for the PR. Maybe mention that it is enabled by default since 0.9.0.1.

Reply:
Thanks for the review. I updated the PR and added a note that it is enabled since 0.9.0.1 and info how to enable it in 0.9.0.